src/Process/PointCloudProcessing.js

```diff
@@ -106,41 +52,18 @@ function initBoundingBox(elt, layer) {
     elt.obj.boxHelper.updateMatrixWorld();
 }
 
-function shouldDisplayNode(context, layer, elt) {
-    let shouldBeLoaded = 0;
-
-    if (layer.octreeDepthLimit >= 0 && layer.octreeDepthLimit < elt.name.length) {
-        return { shouldBeLoaded, surfaceOnScreen: 0 };
+function computeScreenSpaceError(context, layer, elt, distance) {
+    if (distance <= 0) {
+        return layer.sseThreshold;
```

Comment:
is it equivalent to returning Infinity ? (which would be parameterless and thus more explicit)